MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -187,9 +187,11 @@ - (void)start:(NSString *)appSecret withServices:(NSArray<Class> *)services {
       NSMutableArray<NSString*>* servicesNames = [NSMutableArray arrayWithCapacity:sortedServices.count];
 
       for (Class service in sortedServices) {
-        [servicesNames addObject:[self startService:service].storageKey];
+        if ([self startService:service]) {
+          [servicesNames addObject:[service logTag]];
```

Comment:
I saw you added `serviceName` but you don't use that in this line. Please change.